Sources/_AsyncFileSystem/MockFileSystem.swift

```diff
@@ -45,6 +45,14 @@ package actor MockFileSystem: AsyncFileSystem {
         self.storage.content.keys.contains(path)
     }
 
+    /// Writes a sequence of bytes to a file. Any existing content is replaced with new content.
+    /// - Parameters:
+    ///   - path: absolute path of the file to write bytes to.
+    ///   - bytes: sequence of bytes to write to file's contents replacing old content.
+    func write(path: FilePath, bytes: some Sequence<UInt8>) {
```

Comment:
required to fix `MockFileSystem` appending to existing files instead of overwriting on repeated calls to `write`.